cmd/single/start.go

```diff
@@ -122,7 +122,7 @@ func startPropeller(ctx context.Context, cfg Propeller) error {
 			DefaultNamespaces: namespaceConfigs,
 		},
 		NewCache: executors.NewCache,
-		NewClient: executors.NewClient,
+		NewClient: executors.BuildNewClientFunc(50000, propellerScope),
```

Comment:
I think if we don't see the `writeFilterSize` would be different, maybe for now we can just inline it. Wdyt?

Reply:
sounds good to me!